README.md

```diff
@@ -0,0 +1,117 @@
+# Complete integration sample for the Corbado web component
+This is a sample implementation of frontend and backend where the Corbado web component is integrated.
+
+>**Warning**
+>In this tutorial a customer system is built with no preexisting user base, which is effecively realized by having the loginInfo endpoint always return a 404 code when given a username. In case you have an existing user base, you just need to modify this endpoint to return other status codes as well, as described in our [docs](Link wenn online)
+
+## 1. File structure
+    .
+    ├── ...
+    ├── config                        
+    │   └── routes.yaml                 # Assigns paths to controller methods    
+    ├── src                             
+    │   ├── Controller                  
+    │   │   ├── BackendController.php   # Manages endpoints for backend
+    │   └── └── FrontendController.php  # Manages endpoints for frontend
+    ├── templates                     
+    │   ├── home.html.twig              # Home page which you only get to see if you are logged in
+    │   └── login.html.twig             # Login page which contains the Corbado web component
+    └── ...
+
+## 2. How to use
+>**Warning**
+>This sample code corresponds to our [web component integration tutorial](Link wenn online), please read it first in order to understand the flows and business logic!
+
+### 2.1. CNAME
+The only thing you need to create is a CNAME which points to `auth.corbado.com`. We will use `auth.your-company.com` in this tutorial. More info on what a CNAME is and why it is needed can be found in our [docs](https://docs.corbado.com/integrations/web-component#1.-define-cname).
+
+### 2.2. Setup
+
+#### 2.2.1. PHP Symfony
+To start the local development server your system requires [PHP](https://www.php.net/manual/en/install.php) and [Symfony](https://symfony.com/download):
+```
+brew install php
+brew install symfony-cli/tap/symfony-cli
+```
```

Comment:
ist brew aktuell hier der Standard? Kannst du nochmal schauen, wie das andere Tutorials machen. Auch mit composer (gerade wenn man auf anderen Betriebssystemen untwerwegs ist)

Reply:
Die sind standard ja - auf windows is das immer a weng komplizierter, da gibts eigene tutorials nur fürs installieren https://www.geeksforgeeks.org/how-to-install-php-in-windows-10/

Reply:
dann pack den Link gerne dazu, so "For Windows see <URL>"